guides/05_custom-components/01_quick-start.md

```diff
@@ -0,0 +1,120 @@
+# Custom Components in 5 minutes
```

Comment:
Can we rename the actual guide from quick-start.md to something else? Maybe even custom-components-in-5-minutes.md